app/controllers/api/v1/users/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module Users
+      class OmniauthCallbacksController < Devise::OmniauthCallbacksController
+        respond_to :json
+        def google_oauth2
```

Comment:
Please add a line break here
```suggestion
        respond_to :json
 
        def google_oauth2
```